[rospy.client][INFO] 2017-06-09 18:05:13,628: init_node, name[/move_arm_12273_1497056713587], pid[12273]
[xmlrpc][INFO] 2017-06-09 18:05:13,628: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-06-09 18:05:13,632: Started XML-RPC server [http://ip-172-31-19-100:40595/]
[rospy.init][INFO] 2017-06-09 18:05:13,632: ROS Slave URI: [http://ip-172-31-19-100:40595/]
[rospy.impl.masterslave][INFO] 2017-06-09 18:05:13,635: _ready: http://ip-172-31-19-100:40595/
[xmlrpc][INFO] 2017-06-09 18:05:13,645: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-06-09 18:05:13,645: Registering with master node http://localhost:52701
[rospy.init][INFO] 2017-06-09 18:05:13,740: registered with master
[rospy.rosout][INFO] 2017-06-09 18:05:13,744: initializing /rosout core topic
[rospy.rosout][INFO] 2017-06-09 18:05:13,753: connected to core topic /rosout
[rospy.simtime][INFO] 2017-06-09 18:05:13,758: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-06-09 18:05:13,987: topic[/joint_states] adding connection to [http://ip-172-31-19-100:38886/], count 0
[rospy.internal][INFO] 2017-06-09 18:05:13,988: topic[/motion_planning_goal] adding connection to [http://ip-172-31-19-100:37719/], count 0
[rospy.internal][INFO] 2017-06-09 18:05:13,989: topic[/joint_trajectory] adding connection to [/marker_control_9450_1497056458652], count 0
[rospy.internal][INFO] 2017-06-09 18:05:14,895: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-06-09 18:05:14,900: topic[/joint_trajectory] adding connection to [/robot_sim], count 1
[rospy.internal][INFO] 2017-06-09 18:05:14,930: topic[/attached_collision_object] adding connection to [/move_group], count 0
[rospy.internal][INFO] 2017-06-09 18:05:14,935: topic[/collision_object] adding connection to [/move_group], count 0
[rosout][ERROR] 2017-06-09 18:05:20,458: bad callback: <bound method MoveArm.move_arm_cb of <__main__.MoveArm object at 0x7fd9ca9d1390>>
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py", line 720, in _invoke_callback
    cb(msg)
  File "/home/ccc_v1_w_dd4cc_18220/asn13529_Project_5/asn13530_Project_4/work/catkin_ws/src/motion_planning/scripts/motion_planning.py", line 326, in move_arm_cb
    trajectory = self.project_plan(numpy.array(q_start), q_goal, self.q_min, self.q_max)
  File "/home/ccc_v1_w_dd4cc_18220/asn13529_Project_5/asn13530_Project_4/work/catkin_ws/src/motion_planning/scripts/motion_planning.py", line 311, in project_plan
    q_list = self.motion_plan(q_start, q_goal, q_min, q_max)
  File "/home/ccc_v1_w_dd4cc_18220/asn13529_Project_5/asn13530_Project_4/work/catkin_ws/src/motion_planning/scripts/motion_planning.py", line 269, in motion_plan
    if self.is_collision_free(new_node.value, point):
TypeError: is_collision_free() takes exactly 2 arguments (3 given)

[rospy.core][INFO] 2017-06-09 18:06:02,830: signal_shutdown [signal-2]
[rospy.internal][INFO] 2017-06-09 18:06:02,861: topic[/collision_object] removing connection to /move_group
[rospy.internal][INFO] 2017-06-09 18:06:02,864: topic[/joint_trajectory] removing connection to /marker_control_9450_1497056458652
[rospy.internal][INFO] 2017-06-09 18:06:02,868: topic[/joint_trajectory] removing connection to /robot_sim
[rospy.internal][INFO] 2017-06-09 18:06:02,871: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-06-09 18:06:02,876: topic[/attached_collision_object] removing connection to /move_group
[rospy.internal][INFO] 2017-06-09 18:06:02,880: topic[/motion_planning_goal] removing connection to http://ip-172-31-19-100:37719/
[rospy.internal][WARNING] 2017-06-09 18:06:02,881: Unknown error initiating TCP/IP socket to ip-172-31-19-100:47916 (http://ip-172-31-19-100:37719/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2017-06-09 18:06:02,884: Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py", line 307, in close
    c.close()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 807, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.internal][INFO] 2017-06-09 18:06:02,892: topic[/joint_states] removing connection to http://ip-172-31-19-100:38886/
[rospy.impl.masterslave][INFO] 2017-06-09 18:06:02,896: signal-2
[rospy.core][INFO] 2017-06-09 18:06:03,515: signal_shutdown [atexit]
